cfspopcon/formulas/scrape_off_layer/reattachment_models.py

```diff
@@ -0,0 +1,97 @@
+"""Routines to calculate divertor reattachment timescales."""
+
+import numpy as np
+
+from ...algorithm_class import Algorithm
+from ...unit_handling import Unitfull, ureg
+from .two_point_model.momentum_loss_functions import calc_SOL_momentum_loss_fraction
+
+
+@Algorithm.register_algorithm(return_keys=["target_neutral_pressure"])
```

Comment:
Suggest to differentiate between the divertor pressure and the pump duct pressure. Add an extra factor of delta_p_td with a default value of 1.0.